packages/amplify-category-auth/package.json

```diff
@@ -24,6 +24,7 @@
     "amplify-category-function": "2.25.1",
     "amplify-cli-core": "1.3.3",
     "amplify-headless-interface": "1.4.0",
+    "amplify-util-headless-input": "1.3.0",
```

Comment:
it is since this PR without these changes the awscloudformation packages was giving missing dependency during migration, the same change meanwhile added by @edwardfoyle after I discovered it.